provider-example/src/hpke.rs

```diff
@@ -89,6 +106,46 @@ impl Hpke for HpkeRs {
             )
             .map_err(other_err)
     }
+
+    fn setup_opener(
+        &mut self,
+        enc: &EncapsulatedSecret,
+        info: &[u8],
+        secret_key: &HpkePrivateKey,
+    ) -> Result<Box<dyn HpkeOpener + '_>, Error> {
+        let sk_r = hpke_rs::HpkePrivateKey::new(secret_key.secret_bytes().to_vec());
+        let context = self
+            .0
+            .setup_receiver(enc.0.as_slice(), &sk_r, info, None, None, None)
+            .map_err(other_err)?;
+        Ok(Box::new(HpkeRsReceiver { context }))
```

Comment:
Nit: I'd inline the `context` binding here.

Reply:
Fixed.